scripts/constants.js

```diff
@@ -1,16 +1,38 @@
 const path = require('path');
 
+const TYPES = {
+  BASIC_PAGE: 'page',
+  API_DOC: 'api_doc',
+  RELEASE_NOTE: 'release_notes',
+  RELEASE_NOTE_PLATFORM: 'release_notes_platform',
+  TROUBLESHOOTING: 'troubleshooting_doc',
+  WHATS_NEW: 'nr1_announcement',
+  ATTRIBUTE_DEFINITION: 'attribute_definition',
+};
+
 module.exports = {
   BASE_URL: 'https://docs-dev.newrelic.com/api/migration/content',
   BASE_DIR: path.join(__dirname, '..', 'src/content'),
 
-  TYPES: [
-    'page',
-    'api_doc',
-    'release_notes',
-    'release_notes_platform',
-    'troubleshooting_doc',
-    'nr1_announcement',
-    'attribute_definition',
-  ],
+  TYPES,
+
+  GATSBY_CONTENT_TYPES: {
```

Comment:
I'm going to move this to to the frontmatter-specific file since it's not used anywhere else.